File: Do zero ao Hello World/Sprint - 2/ex04.py
import string, random

randomLetters = []
consonantLetters = []
consonants = 0
for i in range(10):
    v = random.choice(string.ascii_letters)
    randomLetters.append(v)
    if v not in ['a', 'A', 'e', 'E', 'i', 'I', 'o', 'O', 'u','U']:
        consonants += 1
        consonantLetters.append(v)

print(randomLetters)
print(f'Foram encontradas {consonants} consoantes. São elas: {consonantLetters}')

File: Do zero ao Hello World/Sprint - 2/ex04.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

func main() {

	rand.Seed(time.Now().Unix())
	letters := [] string{}
	consonants := [] string{}
	count := 0

    //Generate a random character between lowercase a to z
    for i := 0; i < 10; i++ {
		randomChar := 'a' + rune(rand.Intn(26))
		letters = append(letters, string(randomChar))
		switch string(randomChar){
		case "a":
		case "e":
		case "i":
		case "o":
		case "u":
			continue
		default:
			consonants = append(consonants, string(randomChar))
			count += 1
		}
	}
	fmt.Println(letters)
	fmt.Println("Foram encontradas", count, "consoantes. São elas: ", consonants)
}
